target/linux/ar71xx/image/generic-senao.mk

```diff
@@ -39,3 +39,16 @@ define Device/koala
   IMAGE/sysupgrade.bin := append-kernel | pad-to $$$$(KERNEL_SIZE) | append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
 endef
 TARGET_DEVICES += koala
+
+define Device/ursus
+  DEVICE_TITLE := OCEDO Ursus
+  BOARDNAME := URSUS
+  DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca988x
+  KERNEL_SIZE := 1536k
+  IMAGE_SIZE := 7424k
+  MTDPARTS := spi0.0:256k(u-boot)ro,64k(u-boot-env),1536k(kernel),5888k(rootfs),1536k(kernel2),5888k(rootfs2),1088k(data)ro,64k(id)ro,64k(art)ro,7424k@0x50000(firmware)
+  KERNEL := kernel-bin | patch-cmdline | lzma | uImage lzma
```

Comment:
Drop the line, it is the default kernel build command in `target/linux/ar71xx/image/Makefile` anyway.